src/software/ai/hl/stp/play_info.cpp

```diff
@@ -0,0 +1,35 @@
+#include "software/ai/hl/stp/play_info.h"
+
+PlayInfo::PlayInfo() : play_type(""), play_name(""), robot_tactic_assignment({}) {}
+
+PlayInfo::PlayInfo(std::string play_type, std::string play_name,
+                   std::unordered_set<std::string> rta)
```

Comment:
Can you please use a more descriptive parameter name than `rta`? It's fine to reuse `robot_tactic_assignment` here, it will not cause issues with the constructor's initialization list.

**Edit**: Oh I see it's already `robot_tactic_assignment` in the header file. Can we just change that here then?